longevity_test.py

```diff
@@ -290,59 +298,68 @@ def test_user_batch_custom_time(self):
 
         stress_params_list = []
 
-        customer_profiles = self.params.get('cs_user_profiles')
-
         templated_table_counter = itertools.count()
 
-        if customer_profiles:
-            cs_duration = self.params.get('cs_duration')
-            duration = int(cs_duration.translate(str.maketrans('', '', string.ascii_letters)))
-
-            for cs_profile in customer_profiles:
-                cs_profile = sct_abs_path(cs_profile)  # noqa: PLW2901
-                assert os.path.exists(cs_profile), 'File not found: {}'.format(cs_profile)
-                self.log.debug('Run stress test with user profile {}, duration {}'.format(cs_profile, cs_duration))
-
-                user_profile_table_count = self.params.get('user_profile_table_count')  # pylint: disable=invalid-name
-
-                for _ in range(user_profile_table_count):
-                    stress_params_list += self.create_templated_user_stress_params(next(templated_table_counter),
-                                                                                   cs_profile)
-            if not skip_optional_stage('main_load'):
-                self._run_user_stress_in_batches(batch_size=batch_size,
-                                                 stress_params_list=stress_params_list, duration=duration)
+        cs_duration = self.params.get('cs_duration')
+        try:
+            duration = int(cs_duration.translate(str.maketrans('', '', string.ascii_letters))) if cs_duration else None
+        except ValueError:
+            raise ValueError(f"Invalid cs_duration format: {cs_duration}")
+
+        for cs_profile in customer_profiles:
```

Comment:
In your new case it is not customer profile.
So, may be renamed to, for example, `cs_profile`.

Reply:
changed to `cs_user_profiles`